src/Components/Publishing/Typings.ts

```diff
@@ -152,7 +152,7 @@ export interface ImageData {
   url?: string
   partner?: GravityEntity
   slug?: string
-  width?: number
+  width: number
```

Comment:
`height` and `width` can’t really have been optional, otherwise [this division](https://github.com/artsy/reaction/pull/3016/files#diff-fd8c8a460acc6428b745808c9aa18042R74) would always have failed, no?